src/server/auth/server/api_server.go

```diff
@@ -362,6 +362,34 @@ func (a *apiServer) Activate(ctx context.Context, req *auth.ActivateRequest) (re
 	return &auth.ActivateResponse{PachToken: pachToken}, nil
 }
 
+// RotateToken implements the protobuf auth.RotateToken RPC
+func (a *apiServer) RotateAuthToken(ctx context.Context, req *auth.RotateAuthTokenRequest) (resp *auth.RotateAuthTokenResponse, retErr error) {
```

Comment:
We probably want the name to be like `RotateRootToken` to make it clear it only works for the root user.